cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -63,6 +64,62 @@ spec:
             - name: ssl-certs
               mountPath: /etc/ssl/certs
               readOnly: true
+        - image: gcr.io/google_containers/addon-resizer:1.0
+          name: heapster-nanny
+          resources:
+            limits:
```

Comment:
One of the [config best practices](http://kubernetes.io/docs/user-guide/config-best-practices/) is:

> Don’t specify default values unnecessarily, in order to simplify and minimize configs, and to reduce error. 

I don't really understand the argument behind #18216 but it seems to violate that.